pkg/models/cedar_system.go

```diff
@@ -13,3 +13,57 @@ type CedarSystem struct {
 	SystemMaintainerOrgComp string `json:"systemMaintainerOrgComp"`
 	VersionID               string `json:"versionId"`
 }
+
+// BusinessOwnerInformation contains information about the business owner for a CEDAR system
+type BusinessOwnerInformation struct {
```

Comment:
All the non-array fields on this struct need to be using types from the [`null`](https://pkg.go.dev/github.com/guregu/null@v4.0.0+incompatible) or [`null/zero`](https://pkg.go.dev/github.com/guregu/null@v4.0.0+incompatible/zero) packages to indicate that they're optional; right now, go-swagger's converting JSON `null`s into the zero values of the Go types. As an example, for `/system/detail/326-2209-0`, the JSON for `SystemMaintainerInformation` contains:
```
"SystemMaintainerInformation": {
        "ip6EnabledAssetPercent": null,
        "hardCodedIpAddress": null,
        "deploymentFrequency": null,
        // other fields...
}
```
but our GQL returns
```
"systemMaintainerInformation": {
        "ip6EnabledAssetPercent": "",
        "hardCodedIPAddress": false,
        "deploymentFrequency": ""
}
```
I'm not sure if `null` or `null/zero` would be appropriate here, with the way go-swagger's autogenerated code handles things; let's plan on talking about it or pairing on it tomorrow (probably tomorrow afternoon).

Reply:
We decided to not worry about conveying nullability for now; zero values will be fine. See [this Slack message](https://cmsgov.slack.com/archives/CNU2B59UH/p1653588179194699?thread_ts=1653573922.040009&cid=CNU2B59UH). No changes were needed for this PR.